pages/avatarpage.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Ishita Keshawani" />
+  <link rel="stylesheet" href="/styles/avatarpage.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>Avatar</title>
+</head>
+
+<body>
+  <nav class="navbar">
+    <a class="nav-link no-style" href="/index.html">Home</a>
+    <a class="nav-link no-style active" href="./pages/documentation.html">Documentation</a>
+    <i class="far fa-moon nav-link"></i>
+  </nav>
+  <div class="drawer-container">
+    <aside class="aside-component">
+      <div class="sideheader">
+        <h4>Components</h4>
+      </div>
+      <div class="sidebar">
+        <div class="headitems">
+          <div class="item1">
```

Comment:
follow similar naming convention i.e "item1" and other classes have dash in them. so follow same convention  everywhere.